Source/JavaScriptCore/llint/LLIntThunks.cpp

```diff
@@ -535,6 +535,150 @@ MacroAssemblerCodeRef<JSEntryPtrTag> fuzzerReturnEarlyFromLoopHintThunk()
     return codeRef;
 }
 
+#if ENABLE(JIT)
+MacroAssemblerCodeRef<JITThunkPtrTag> arityFixupThunk()
+{
+    static LazyNeverDestroyed<MacroAssemblerCodeRef<JITThunkPtrTag>> codeRef;
+    static std::once_flag onceKey;
+    std::call_once(onceKey, [&]{
+        CCallHelpers jit;
+
+        // We enter with fixup count in argumentGPR0
+        // Caller's linkRegister in argumentGPR1
```

Comment:
Can you add `(preserved for caller)` here since argumentGPR1 isn't used directly in `arityFixupThunk`